src/constants.tsx

```diff
@@ -1,27 +1,41 @@
+import {assertNever} from "./utils";
+
 export const devnetUrl =
   process.env.APTOS_DEVNET_URL || "https://fullnode.devnet.aptoslabs.com/";
 
-export const networks: Record<string, string> = {
+export const networks = {
   local: "http://localhost:8080",
   devnet: devnetUrl,
   test: "https://rosetta.aptosdev.com/",
-  //governance: "https://ait2.aptosdev.com/", // TODO - use the proposals url
 };
 
-export const walletNetworkMap: Record<string, string> = {
-  Devnet: "devnet",
-  Localhost: "local",
-  Testnet: "test",
+export const walleExplorertNetworkMap = (
+  walletNetwork: WalletNetworks,
+): ExplorerNetworks => {
+  switch (walletNetwork) {
+    case "Devnet":
+      return "devnet";
+    case "Localhost":
+      return "local";
+    case "Testnet":
+      return "test";
+    default:
+      return assertNever(walletNetwork);
+  }
 };
 
+export type ExplorerNetworks = keyof typeof networks;
```

Comment:
For consistency with the rest of the code, it should probably be called `NetworkName`.